Unix/samples/Providers/TestClass_AllDMTFArrays/TestClass_AllDMTFArrays.cpp

```diff
@@ -119,14 +120,15 @@ void SetUint32Array(TestClass_AllDMTFArrays *instance, MI_Uint32 count)
     {
         for(MI_Uint32 i=0; i < count; i++)
         {
-            uint32Array[i] =  (MI_Uint32) i % 4294967296;
+            uint32Array[i] =  (MI_Uint32) i % 0xFFFFFFFF;
```

Comment:
Yeah, it will be better: ULONG_MAX->4294967295 , LONG_MAX->2147483647 , LLONG_MAX ->9223372036854775807 , ULLONG_MAX ->18446744073709551615 .